chatlas/_tokens.py

```diff
@@ -71,17 +87,89 @@ def tokens_reset() -> None:
     _token_counter = ThreadSafeTokenCounter()
 
 
+class TokenPrice(TypedDict):
+    """
+    Defines the necessary information to look up pricing for a given turn.
+    """
+
+    provider: str
+    """The provider name (e.g., "OpenAI", "Anthropic", etc.)"""
+    model: str
+    """The model name (e.g., "gpt-3.5-turbo", "claude-2", etc.)"""
+    cached_input: float
+    """The cost per user token in USD per million tokens for cached input"""
+    input: float
+    """The cost per user token in USD per million tokens"""
+    output: float
+    """The cost per assistant token in USD per million tokens"""
+
+
+# Load in pricing pulled from ellmer
+f = resources.files("chatlas").joinpath("data/prices.json").read_text(encoding="utf-8")
+PricingList: list[TokenPrice] = orjson.loads(f)
```

Comment:
PascalCase usually signals that this is a type -- I think I'd prefer snake_case here